compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/util/Constants.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.ballerinalang.compiler.util;
+
+/**
+ * Class that holds constant values.
+ *
+ * since 0.980.0
+ */
+public class Constants {
+
+    private Constants() {
+    }
+
+    public static final int OPEN_SEALED_ARRAY_INDICATOR = -2;
+    public static final int UNSEALED_ARRAY_INDICATOR = -1;
```

Comment:
Shall we move this to BLangConstants class?

Reply:
This class is in ballerina-core and compiler has no dependency to it.